### ISSUE TITLE
.NET Core 3.0.0 Preview 5

### ISSUE THREAD

**Issue by vivmishra** [dotnet/core#2685]
[Release Notes](https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/3.0.0-preview5.md)
[Download](https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/3.0.0-preview5-download.md)
[Blog post](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0-preview-5/)

Please report any issues you find with 3.0.0 Preview 5, either responding to this issue, creating a new issue or creating a new issue in one of the following repos:

* [aspnet/Home](https://github.com/aspnet/Home) - for ASP.NET Core questions
* [aspnet/EntityFrameworkCore](https://github.com/aspnet/EntityFrameworkCore) - for Entity Framework Core questions
* [dotnet/cli](https://github.com/dotnet/cli/issues) - for CLI tools and questions
* [dotnet/corefx](https://github.com/dotnet/corefx/issues) - for API issues and questions
* [dotnet/coreclr](https://github.com/dotnet/coreclr/issues) - for runtime issues
* [dotnet/core-setup](https://github.com/dotnet/core-setup/issues) - for installer issues
* [dotnet/winforms](https://github.com/dotnet/winforms/issues) - for Winforms issues
* [dotnet/wpf](https://github.com/dotnet/wpf/issues) - for WPF issues
* [nuget/home](https://github.com/nuget/home/issues) - for NuGet questions and issues

**Comment by michael-hawker** [comment]
I can't seem to install preview 5?  I keep getting a message about another install occurring, but I'm not doing anything on my box, even after a clean boot:

![image](https://user-images.githubusercontent.com/24302614/57275112-47752700-7052-11e9-80b2-570c6e9cb41a.png)

WinOS: 17763.437

Log: 
[Microsoft_.NET_Core_SDK_3.0.100_-_preview5_(x64)_20190506225432.log](https://github.com/dotnet/core/files/3150995/Microsoft_.NET_Core_SDK_3.0.100_-_preview5_.x64._20190506225432.log)

**Comment by michael-hawker** [comment]
Actually, just tried preview 4 and had the same issue... 😐

Log looks similar.  Any ideas on error codes and what I can try to unblock install?

**Comment by vivmishra** [comment]
@peterhuene @livarcocc

**Comment by peterhuene** [comment]
@johnbeisner Have you seen this MSI error before from our installer?

```
[70F8:0690][2019-05-06T22:54:40]e000: Error 0x80070652: Failed to install MSI package.
[70F8:0690][2019-05-06T22:54:40]e000: Error 0x80070652: Failed to execute MSI package.
[5EB8:706C][2019-05-06T22:54:40]e000: Error 0x80070652: Failed to configure per-machine MSI package.
[5EB8:706C][2019-05-06T22:54:40]w348: Application requested retry of package: dotnet_runtime_3.0.0_preview5_27626_15_win_x64.msi, encountered error: 0x80070652. Retrying...
[70F8:0690][2019-05-06T22:54:43]i301: Applying execute package: dotnet_runtime_3.0.0_preview5_27626_15_win_x64.msi, action: Install, path: C:\ProgramData\Package Cache\{9D024CE3-2F96-4682-8A31-DB6ADCE181C1}v24.0.27626\dotnet-runtime-3.0.0-preview5-27626-15-win-x64.msi, arguments: ' ARPSYSTEMCOMPONENT="1" DOTNETHOME="C:\Program Files\dotnet"'
```

**Comment by johnbeisner** [comment]
I've installed the .Net core SDK 3.0.100-preview5-011568 from the bundle installer and could not reproduce the issue.

**_Can we look at the installation log for the dotnet_runtime_3.0.0_preview5_27626_15_win_x64.msi_**

It's name will be of the pattern: 
Microsoft_.NET_Core_SDK_3.0.100_-_preview5_(x64)_20190507125005_000_dotnet_runtime_3.0.0_preview5_27626_15_win_x64.msi.log

Windows can be updating in the background. The tipoff is that 2 MSIEXEC processes are running when you are not doing anything on the box..

**Comment by michael-hawker** [comment]
![image](https://user-images.githubusercontent.com/24302614/57334203-ec841400-70d3-11e9-9d6c-c8b6b4c19925.png)

I've been trying again, but the installer just seems to be getting stuck now half-way through.  The most recent msi log is here:

[Microsoft_.NET_Core_SDK_3.0.100_-_preview5_(x64)_20190507141946_000_dotnet_runtime_3.0.0_preview5_27626_15_win_x64.txt](https://github.com/dotnet/core/files/3154606/Microsoft_.NET_Core_SDK_3.0.100_-_preview5_.x64._20190507141946_000_dotnet_runtime_3.0.0_preview5_27626_15_win_x64.txt)

**Comment by joeloff** [comment]
I did notice: MsiSystemRebootPending is being set to 1 in the log, that could indicate a pending reboot due to a file rename that occurred. The log seems to just end, which make me wonder whether whatever is causing it to hang, is causing the bundle to timeout and retry, which would then explain the "another install in progress error" because the previous MSIEXEC is still active and haven't technically ended

**Comment by michael-hawker** [comment]
I just booted my machine this morning and the installer popped up right away, I clicked install and it went successfully... so confused, but unblocked?

**Comment by joeloff** [comment]
@michael-hawker that's probably because there was a pending reboot. As I mentioned previously, the MSI log seemed to show that there was a pending reboot. Once you restart it should resume (and sounds like it did).

**Comment by michael-hawker** [comment]
Yeah, interesting, because I had tried a shut-down and restart in between at one point and had the same problem, so something else must have jumped in each time until they were all cleared out.

**Comment by karelz** [comment]
Superseeded by Preview 6 - #2859
